### PR TITLE
refactor(ui): dedupe bond types

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Remove duplicate bond types.

## QA

N/A

## Fixes

Fixes: #2351.

### CODE REVIEW

File: ui/src/app/store/general/selectors/bondOptions.ts [hunk 1]
```diff
@@ -10,23 +10,22 @@ const generalSelectors = generateGeneralSelector<"bondOptions">("bondOptions");
 /**
  * Returns a shallow/deduped array of LACP rates.
  */
-const lacpRates = createSelector([generalSelectors.get], ({ lacp_rates }) =>
-  lacp_rates.map(([lacpRate]) => lacpRate)
+const lacpRates = createSelector([generalSelectors.get], (options) =>
+  options?.lacp_rates.map(([lacpRate]) => lacpRate)
```
Comment (huwshimi) [inline ui/src/app/store/general/selectors/bondOptions.ts:14]: Drive-by fixes for bond types that haven't loaded yet.

**Review by Caleb-Ellis — APPROVED** [review]
